<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="kangsta" name="searchitems.SearchItemsWithDatabaseDataSource" tests="1" failures="0" timestamp="27 May 2017 19:01:36 GMT" time="11.555" errors="1">
  <testcase name="searchItemsWithDB" time="11.555" classname="searchitems.SearchItemsWithDatabaseDataSource">
    <error type="java.io.FileNotFoundException" message="src\MySql.properties (The system cannot find the file specified)">
      <![CDATA[java.io.FileNotFoundException: src\MySql.properties (The system cannot find the file specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileInputStream.<init>(FileInputStream.java:93)
	at utility.reader.ConnectDB.loadProperties(ConnectDB.java:31)
	at utility.reader.ConnectDB.connectToDatabase(ConnectDB.java:39)
	at utility.reader.ConnectDB.readDataBase(ConnectDB.java:85)
	at datasource.DatabaseDataReader.dataFromDatabase(DatabaseDataReader.java:20)
	at searchitems.SearchItemsWithDatabaseDataSource.searchItemsWithDB(SearchItemsWithDatabaseDataSource.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </error>
  </testcase> <!-- searchItemsWithDB -->
</testsuite> <!-- searchitems.SearchItemsWithDatabaseDataSource -->
